Cargo.toml

```diff
@@ -3,6 +3,8 @@ members = [
     "serde",
     "serde_derive",
     "serde_derive_internals",
-    "serde_test",
     "test_suite",
 ]
+
+[patch.crates-io]
+serde = { path = "serde" }
```

Comment:
This PR breaks any attempts to experiment with changing serde public API. I think, it is due to this line. Is it possible to do without it? What is it for?